src/schedule.py

```diff
@@ -8,3 +15,146 @@ def add_match(self, match):
 
     def get_matches(self, match):
         self.teams_matches.append(match)
+
+    @staticmethod
+    def from_json_file(filename: str):
+        data = ''
+        with open(filename) as files:
+            text = files.read()
+            data = json.loads(text)
+
+        matches = list()
+        for match_info in data:
+            m = match.Match(match_info["Match"], match_info["Location"], match_info["Date"], match_info["Team 1"],
+                            match_info["Team 2"], [], [])
+            matches.append(m)
+
+        schedule = Schedule(matches)
+        return schedule
+
+    def get_match(self, number):
+        for m in self.matches:
+            if m.number == str(number):
+                return m
+
+    def play_match(self, number_of_match, teams):
+        m = self.get_match(number_of_match)
+        host_name, gest_name = m.host, m.guest
+        host_rating = 1
+        gest_rating = 1
+        print(f'Сегодня, на стадионе {m.stadium} состоится {m.number} матч Чемпионата мира по футболу 2022')
+        print(f'В рамках данного матча встречаются команды {host_name} и {gest_name}')
+        print()
+        for t in teams:
+            if host_name == t.name:
+                host_team = t
+                host_rating = t.get_rating(m, teams)
+                print(f'Стартовый состав команды хозяев {host_name}:')
+                t.first11_presentation(m, teams)
+                print()
+            if gest_name == t.name:
+                gest_team = t
+                gest_rating = t.get_rating(m, teams)
+                print(f'Стартовый состав команды  гостей {gest_name}:')
+                t.first11_presentation(m, teams)
+
+        print(f'рейтинг команд: {host_name} - {host_rating}   :    {gest_name} - {gest_rating}')
+        print()
+
+        chance = ev.ChanceState()
```

Comment:
It seems to be irrelevant here because class Event doesn't have such methods. I would be better to do git merge from develop to see current the develop branch.